### ISSUE TITLE
[Design] Personal information revision: Add "why do we ask for this?" finalized copy to each field

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#36240]
### Background

In user research, we learned that people didn't understand why we asked for some of the personal information fields, and they were concerned about how their information would be used. 

We need to make it clear 1) why we ask for each field, and 2) that each of these new fields is optional, and people do not need to share this information. We need to be especially clear with caveats around people disclosing whether they are transgender or not and with sharing sexual orientation.

### Tasks

Work with Samara/Danielle as needed to do the following:

- [x]  Reach out to Lexi to find out more about why we ask for each of the new fields, and how that information will be used
- [x] Talk to Danielle about the "orphaned" words at the ends of each line when it wraps
- [x] Update the hint text copy on each field
- [x] Update the "Why do we ask for this?" copy for preferred name
- [x] Update the "Why do we ask for this?" copy for pronouns
- [x] Update the "Why do we ask for this?" copy for gender identity
- [x] Update the "Why do we ask for this?" copy for sexual orientation
- [x] Make it clear all of these fields are optional
- [ ] Add a link to a Resources & Support page that provides more detailed info
- [ ] All updates have been reviewed and agreed upon by Christina/Danielle/Marci/Samara

### Acceptance criteria

- [x]  Reach out to Lexi to find out more about why we ask for each of the new fields, and how that information will be used
- [x] Talk to Danielle about the "orphaned" words at the ends of each line when it wraps
- [x] Update the hint text copy on each field
- [x] Update the "Why do we ask for this?" copy for preferred name
- [x] Update the "Why do we ask for this?" copy for pronouns
- [x] Update the "Why do we ask for this?" copy for gender identity
- [x] Update the "Why do we ask for this?" copy for sexual orientation
- [ ] Make it clear all of these fields are optional
- [ ] Add a link to a Resources & Support page that provides more detailed info
- [ ] All updates have been reviewed and agreed upon by Christina/Danielle/Marci/Samara

**Comment by Samara-Strauss** [comment]
I reached out to Lexi just now for more information. I'll share it here as we have it.

**Comment by MarciMcGuireGCIO** [comment]
Making note that the info specifying that the fields are not required will be part of the "why we ask for this" help text and not part of the label or the help text itself.

**Comment by Samara-Strauss** [comment]
For clarity -- this ticket is for Christina to update mocks. Danielle is working on the actual copy in #36862, so any conversation on copy will happen there. Also, adding the `blocked ` label to this since it is dependent on #36862.

**Comment by MarciMcGuireGCIO** [comment]
@cgednalske - just want to confirm we are moving this to Sprint 68, correct?

**Comment by cgednalske** [comment]
@MarciMcGuireGCIO correct! I can now get started with some of the points Danielle addressed, but they are significant changes + whatever comes out of the new content items.

**Comment by MarciMcGuireGCIO** [comment]
Splitting last two tasks out into new content ticket #38126 due to unknowns around MPI's ability to support remove/delete button.
